contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -166,6 +166,7 @@ script: |
     rm -rf ${DISTNAME}/lib/pkgconfig
     find ${DISTNAME}/bin -type f -executable -print0 | xargs -0 -n1 -I{} ../contrib/devtools/split-debug.sh {} {} {}.dbg
     find ${DISTNAME}/lib -type f -print0 | xargs -0 -n1 -I{} ../contrib/devtools/split-debug.sh {} {} {}.dbg
+    cp ../doc/README.md ${DISTNAME}/
```

Comment:
in commit 503a067f105cbfd249fd852a66569799e980f123:

why?

the doc readme has a ton of internal links, which are all dead (obviously) when shipped in the zip

Reply:
Should that be removed from the `gitian-linux.yml` as well?

Reply:
This has been added in 8550f1fb28292a081aab3f49ba2fb561710c4572 by @hebasto as a fix for #8160. 

Though, I think that instead of ./doc/README.md, it should say ./README.md (at the root)

Reply:
Very true, will fix for both.

Reply:
I agree that the current README's are pretty much completely useless to package with a binary distribution. I don't think #8160 ever really was solved.

Note that for Windows specifically there is `doc/README_windows.txt` as user documentation meant to be copied to the distributions as `readme.txt`.
The NSI installer does this, but the descriptor for the tarball doesn't.

Reply:
@laanwj should I do
```
cp ../doc/README_windows.txt ${DISTNAME}/readme.txt
```
instead?

Reply:
yes, that's what i'd suggest, it's the most appropriate file for this